tests/test_api.py

```diff
@@ -155,8 +157,8 @@ def test_to_from_bytes(self):
             self.assertEqual(metadata_obj_2.to_bytes(), obj_bytes)
 
     def test_sign_verify(self):
-        root_path = os.path.join(self.repo_dir, "metadata", "root.json")
-        root = Metadata[Root].from_file(root_path).signed
+        path = os.path.join(self.repo_dir, "metadata", "root.json")
+        root = Metadata[Root].from_file(path).signed
```

Comment:
Above you changed from ``dir`` to ``dest_dir``, I see it as more explicit.
At these lines, I see the ``root_path`` as ``path``, any specific reason for this?

Reply:
dir was likely changed because it's a builtin, this change is likely to keep lines below 80 chars

Reply:
`root_path` was changed to `path`, because there was a pylint warning we were using too many variables in a single 
function and with this change, we can reuse `path`.
About `dir` Jussi is correct.

Reply:
re-using variables is usually a code smell but this looks OK to me. The best solution might be a refactor of the function (to maybe multiple tests) but I appreciate not doing large changes under "address pylint warnings": anyway LGTM

Reply:
This is no longer of a concern after the merge of #1671 and I removed this change.